src/jitlayers.cpp

```diff
@@ -1225,6 +1226,7 @@ JuliaOJIT::JuliaOJIT()
 #endif
     GlobalJD(ES.createBareJITDylib("JuliaGlobals")),
     JD(ES.createBareJITDylib("JuliaOJIT")),
+    libgccJD(ES.createBareJITDylib("Julialibgcc")),
```

Comment:
Call it JuliaCompilerRT?  And crtJD?

Reply:
I gave them that name because I wanted to see if doing that would work. But it seems windows is just broken.
Win64 probably has the same error, but hangs instead of failing. Not sure what to do now.